.taskcluster.yml

```diff
@@ -6,329 +6,314 @@ policy:
-                                        then:
-                                            MOZILLAVPN_PULL_REQUEST_NUMBER: '${event.pull_request.number}'
-                                      - $if: 'tasks_for == "action"'
-                                        then:
-                                            ACTION_TASK_GROUP_ID: '${action.taskGroupId}'  # taskGroupId of the target task
-                                            ACTION_TASK_ID: {$json: {$eval: 'taskId'}}  # taskId of the target task (JSON-encoded)
-                                            ACTION_INPUT: {$json: {$eval: 'input'}}
-                                            ACTION_CALLBACK: '${action.cb_name}'
-                                      - $if: 'tasks_for == "github-release"'
-                                        then:
-                                            MOZILLAVPN_HEAD_TAG: '${event.release.tag_name}'
-                              features:
-                                  taskclusterProxy: true
-                                  chainOfTrust: true
-                              # Note: This task is built server side without the context or tooling that
-                              # exist in tree so we must hard code the hash
-                              image:
-                                  mozillareleases/taskgraph:decision-e035c4bb24e3a05bcd666518756aa69fac302ed1e73fd39d39cd182fc4470818@sha256:6b0be618c3909d50bd059d2451090f1a8f36d64f0bcddbcc32f1559c5ebbee58
+                    # Note: This task is built server side without the context or tooling that
+                    # exist in tree so we must hard code the hash
+                    image: mozillareleases/taskgraph:decision-cf4b4b4baff57d84c1f9ec8fcd70c9839b70a7d66e6430a6c41ffe67252faa19@sha256:425e07f6813804483bc5a7258288a7684d182617ceeaa0176901ccc7702dfe28
 
-                              maxRunTime: 1800
+                    maxRunTime: 1800
 
-                              command:
-                                  - /usr/local/bin/run-task
-                                  - '--mozillavpn-checkout=/builds/worker/checkouts/src'
-                                  - '--taskgraph-checkout=/builds/worker/checkouts/taskgraph'
-                                  - '--task-cwd=/builds/worker/checkouts/src'
-                                  - '--'
-                                  - bash
-                                  - -cx
-                                  - $let:
-                                        extraArgs: {$if: 'tasks_for == "cron"', then: '${cron.quoted_args}', else: ''}
-                                    in:
-                                        $if: 'tasks_for == "action"'
-                                        then: >
-                                            PIP_IGNORE_INSTALLED=0 pip3 install --user /builds/worker/checkouts/taskgraph &&
-                                            PIP_IGNORE_INSTALLED=0 pip3 install --user mozilla-version &&
```

Comment:
This wasn't added to the `requirements.in`, is it unused?

Reply:
For posterity: `mozilla-version` wasn't used anywhere.